config/config_test.go

```diff
@@ -77,7 +77,7 @@ func TestLoadConfig(t *testing.T) {
 	testDataPath := filepath.Join(currentDir, "/testdata/test_config.toml")
 
 	var config Config
-	LoadConfig(ViperParams{
+	_ = LoadConfig(ViperParams{
```

Comment:
[q] I'm assuming this is the linter complaining about ignored `err` returns?

Reply:
Yup exactly 😄